app/assets/stylesheets/fullcalendar_engine/style.css

```diff
@@ -17,7 +12,3 @@ body{
 span > div#loading {
   position: fixed;
 }
-
-h1{
-font-size: 16px;
-}
```

Comment:
That's the problem. Obviously, it is possible that someone is using that as default CSS. It is also possible that others (me included) do not want these CSS rules and has overwritten them (or forked the repo and removed them as I did).

The same goes for the rules for `body`, right?

You don't want to rely on an engine setting CSS rules. What should we do about it?